python/ooi_data_explorations/uncabled/process_nutnr.py

```diff
@@ -147,12 +148,13 @@ def suna_datalogger(ds, burst=True):
 
     # address incorrectly set units and variable types
     ds['dark_value_used_for_fit'].attrs['units'] = 'counts'
-    ds['serial_number'] = ('time', [int(''.join(x.astype(str))) for x in ds.serial_number.data])
-    ds['serial_number'].attrs = dict({
-        'long_name': 'Serial Number',
+    # CGSN update: switched to simple type conversions
```

Comment:
Need this, and the `serial_number` below, reset back to what I had. There were a small number of files in the GC and downloaded via M2M where, for whatever reason, the serial number and frame type got split apart into a 2D array instead of a 1D array. The simple type conversion you put back in is what I had before, and it works with the 1D arrays, but that would fail when it it hit the 2D arrays.

Reply:
I had changed that because it was throwing an Error for me with the anonymous function. I added a if-else to check if the shape is 1 or 2-d, and then operate. This worked for me just as well.